src/TextUI/Help.php

```diff
@@ -100,7 +100,7 @@ final class Help
             ['arg'    => '--printer <printer>', 'desc' => 'TestListener implementation to use'],
             ['spacer' => ''],
 
-            ['arg'  => '--order-by=<order>', 'desc' => 'Run tests in order: default|reverse|random|defects|no-depends'],
+            ['arg'  => '--order-by=<order>', 'desc' => 'Run tests in order: default|reverse|random|defects|no-depends|duration'],
```

Comment:
Do you have any preferences in regard to the order 

* in which the options are listed in the help output
* in which the options are handled in the `switch`-statement

?

Personally, I would probably keep them sorted like this

* `default` first
* everything else alphabetically

If you like this, I will adjust!

Reply:
I like :)

Reply:
Nice addition :)

Reply:
Thanks, adjusted!